src/opencensus/proto/trace/v1/trace.proto

```diff
@@ -290,6 +291,11 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 11;
 
+  // An optional resource that this span describes. If not set, this span 
```

Comment:
span doesn't describe resource. May be this should be rephrased to something like "An optional resource to which this span is associated with".

Reply:
Reworded. Please take another look